src/components/ActualStepDescription/ActualStepDescriptionComponent.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { HtmlElementProps } from "../../utlis";
+
+interface ActualStepDescriptionProps extends HtmlElementProps<HTMLDivElement> {}
+
+function ActualStepDescriptionF(
+  props: ActualStepDescriptionProps,
+  ref?: React.Ref<HTMLDivElement>
+): JSX.Element {
+  return (
+    <div {...props} ref={ref}>
+      Actual step description goes here
+    </div>
+  );
+}
+
+export const ActualStepDescription = React.forwardRef(ActualStepDescriptionF);
```

Comment:
Jakbyście się zastanawiały nad tym do czego to jest potrzebne i dlaczego tak często się pojawia.
`Ref` to jest mechanizm React'a który przechowywać referencję do konkretnego obiektu. Przykłądem użycia może być interakcja wprowszt z elemetami HTMLowymi. (czyli np. zmiana szerokości/wysokości, dodawania styli, odczytawanie wymiarów elementu oraz jego zawartości i inne "niskopoziomiowe" rzeczy. 

 Użycie Ref'a [nie jest  zalecane przez twórców React'a](
https://reactjs.org/docs/forwarding-refs.html)
ale nie mniej jednak czasami jest przydatne. 
W tym przypadku jest użyłem tego, ponieważ taki był wymóœ biblioteki `react-grid-layout` z której skorzystałem.